internal/configuration/test_resources/config_oidc_modern.yml

```
@@ -262,6 +273,6 @@ identity_providers:
         - 'https://example.com'
     clients:
       - client_id: 'abc'
-        client_secret: '123'
+        client_secret: '$pbkdf2-sha512$310000$c8p78n7pUMln0jzvd4aK4Q$JNRBzwAo0ek5qKn50cFzzvE9RXV88h1wJn5KGiHrD0YKtZaR/nCb2CJPOsKaPK0hjf.9yHxzQGZziziccp6Yng'
```

```suggestion
        client_secret: '$pbkdf2-sha512$310000$c8p78n7pUMln0jzvd4aK4Q$JNRBzwAo0ek5qKn50cFzzvE9RXV88h1wJn5KGiHrD0YKtZaR/nCb2CJPOsKaPK0hjf.9yHxzQGZziziccp6Yng' # yamllint disable-line rule:line-length
```
Is this a hash of `insecure_secret` or something else?

---

Whichever one we use in OIDC docs.